.github/workflows/conformance-aks-v1.10.yaml

```diff
@@ -53,10 +53,10 @@ concurrency:
     ${{
       (github.event_name == 'schedule' && github.sha) ||
       (github.event_name == 'issue_comment' && (
-        startsWith(github.event.comment.body, 'ci-aks-1.10') ||
-        startsWith(github.event.comment.body, '/ci-aks-1.10') ||
-        startsWith(github.event.comment.body, 'test-backport-1.10') ||
-        startsWith(github.event.comment.body, '/test-backport-1.10')
+        github.event.comment.body == 'ci-aks-1.10' ||
```

Comment:
Doing an exact match here will break if there's a newline after `/ci-aks-1.0`. Can we use a regexp match instead?

Reply:
It seems that [there's no regexp matching available](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#functions) 😢

Reply:
As `/test` is the problem, how about for `/test` only:

```
(startsWith(github.event.comment.body, '/test') && !startsWith(github.event.comment.body, '/test-'))
```

it's a hack, but is resistant to new lines and comments after the `/command`.

Reply:
Yea, as said in the commit message my main goal was getting rid of the "double testing" issue, with this limitation as side-effect. Unfortunately GitHub Actions does not support filtering strings using regex (see [here](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions) for what's possible in native syntax), so even if we wanted to we couldn't do it from within the `if` context. Would have to kludge around with a shell step actively failing the job if not matching...

Reply:
You posted so fast GH API did not have time to show me the messages, wow! :D
OK let's ship it.